app/lib/flash_alerts.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module FlashAlerts
+  extend ActiveSupport::Concern
+
+  ALERT_TYPES = %i[default info warning success danger].freeze
```

Comment:
Changing the string constants to symbol. I noticed that the default constants (`:notice`, `:error`, etc.) are symbols, so mixing them causes quite some confusion.